app/_hub/kong-inc/openid-connect/how-to/_fapi.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Financial-Grade API (FAPI)
+nav_title: Financial-Grade API (FAPI)
+minimum_version: 3.7.x
+---
+
+
+The OpenID Connect plugin supports various features of the FAPI standard, aimed to protect APIs that expose high-value and sensitive data.
+
+### Pushed Authorization Requests (PAR)
+
+With PAR enabled, Kong (OAuth client) sends the payload of an authorization request to the IdP. As a result, it obtains a `request_uri` value. This value is used by the client in a call to the authorization endpoint as a reference to obtain the authorization request payload data.
+
+For more information refer to the [Configuration reference](/hub/kong-inc/openid-connect/configuration/#config-pushed_authorization_request_endpoint)
```

Comment:
```suggestion
For more information, refer to the [configuration reference for the `pushed_authorization_request_endpoint` parameter](/hub/kong-inc/openid-connect/configuration/#config-pushed_authorization_request_endpoint).
```
It's nice to include the config param name here so that they can skim the page.